demo-extended/src/main/java/com/novoda/simplechromecustomtabs/demo/linkify/WebLinkify.java

```diff
@@ -0,0 +1,59 @@
+package com.novoda.simplechromecustomtabs.demo.linkify;
+
+import android.text.Spannable;
+import android.text.SpannableString;
+import android.text.Spanned;
+import android.text.method.LinkMovementMethod;
+import android.text.style.URLSpan;
+import android.widget.TextView;
+
+import java.util.List;
+
+/**
+ * A custom version of {@link android.text.util.Linkify} which allows the user to set their own listener for when any of the Web links has been clicked
+ */
+public final class WebLinkify {
```

Comment:
Not quite happy with the name, but I couldn't think of a name that says:
1. this is a custom version of Android's Linkify
2. It works for Web links only
3. What it basically does is to allow you to set a custom click listener

suggestions welcome! :dancer: